.gitignore

```diff
@@ -3,6 +3,7 @@
 javadoc/
 docs/source/casual-docs/
 docs/source/version
+.qodo/
```

Comment:
? codium? Riiiiight, now it clicks what's up with this PR!

Reply:
ha ha, I haven't used that for this no but yeah, I have played with the java code to see what it can do :)